tests/zfs-tests/tests/functional/rsend/rsend_019_pos.ksh

```diff
@@ -36,7 +36,7 @@
 verify_runnable "both"
 
 # See issue: https://github.com/zfsonlinux/zfs/issues/6086
-if is_linux; then
+if is_32bit; then
```

Comment:
There have been reports that this test, and other rsend tests, are occasionally failing due to low default `zfs_arc_min` value.  We may want to consider increase the minimum value slightly, although the fixes in 787acae0b5cd139ea0f9fa60558cca28d4673b23 may make that unnecessary.

Reply:
I need this test enabled to push https://github.com/zfsonlinux/zfs/pull/6623 forward: i wonder if dumping the send streams in filesystems like `/$sendpool` and `/$streamfs` is really needed or is just stressing ZFS more than necessary for rsend tests.

Reply:
I'm all for enabling this test case as long as we can run it reliably.

I've opened PR #6659 which increases the default `arc_c_min` on systems which have more then 1G of total system memory.  It will be set to 1/32 of total system memory with the existing 32M floor preserved.  This brings the code a little more inline with OpenZFS although we still permit a lower floor in order to support low memory systems.

It also means that for the buildbot TESTERS `arc_c_min` will now default to 128M (1/32 of 4G) which based open issues is expected to be large enough to avoid the reported performance with the rsend tests.

As for dumping the send streams in `/$sendpool` and `/$streamfs` that may put further stress on the system but that's exactly the kind of thing we want to test.  Let's leave it as is unless we determine we have to change it.